src/config/editorsImgMap.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  ANDROIDSTUDIO,
+  ATOM,
+  CHROME,
+  ECLIPSE,
+  SUBLIMETEXT2,
+  SUBLIMETEXT3,
+  VIM,
+  VSCODE,
+  XCODE
+} from "../constants/editors";
+import {
+  androidStudio128Path,
+  atom128Path,
+  chrome128Path,
+  eclipse128Path,
+  sublimeText128Path,
+  vim128Path,
+  vsCode128Path,
+  xcode128Path
+} from "../constants/imgPaths";
+
+interface EditorImageMap {
+  [s: string]: string;
+}
+
+const imgMap: EditorImageMap = {
+  [ANDROIDSTUDIO]: androidStudio128Path,
+  [ATOM]: atom128Path,
+  [CHROME]: chrome128Path,
+  [ECLIPSE]: eclipse128Path,
+  [SUBLIMETEXT2]: sublimeText128Path,
+  [SUBLIMETEXT3]: sublimeText128Path, // Should this be a different image?
```

Comment:
I'm wondering if we should solve for editor versions with different plugin scopes, because other editors like Visual Studio can have WakaTime installed in one version but not the other. So combining Sublime into one editor with two installable versions.

Reply:
[I think we should put versions in this object](https://github.com/wakatime/desktop/pull/32/files#diff-7618a0f6fa75d7ac47adb361bcad3c0dR5). It is a bit bare-bones right now, but I think versions should go in that object. Then combine SUBLIMETEXT2/3 together under one constant.